src/EventListener/ExceptionListener.php

```diff
@@ -23,8 +24,15 @@
  * @author Samuel ROZE <samuel.roze@gmail.com>
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-final class ExceptionListener extends BaseExceptionListener
+final class ExceptionListener
 {
+    private $exceptionListener;
+
+    public function __construct($controller, LoggerInterface $logger = null, $debug = false, string $charset = null, $fileLinkFormat = null)
+    {
+        $this->exceptionListener = new BaseExceptionListener($controller, $logger, $debug, $charset, $fileLinkFormat);
```

Comment:
@meyerbaptiste In which case would the constructor have these arguments? I can't find anything in Symfony.

Reply:
Oh, looks like it was reverted: https://github.com/symfony/symfony/pull/31868

So we don't need to handle that.